framework/resources/Functional/table_function/positive/plan/drill-5166_npe_with_table_function_group_select_plan_check.e

```diff
@@ -0,0 +1 @@
+UnionExchange
```

Comment:
Why are we only checking for UnionExchange in all tests?

Reply:
I need to check that more than one major fragments are being created. That was the case when the null pointer exception happened on previous releases. With only one fragment other tests will definitely pass but they will verify nothing.

Reply:
Got it, thanks.